src/Network/Benchmark.hs

```diff
@@ -67,9 +67,10 @@ import Control.Lens hiding ((.=))
 import Control.Monad
 import Control.Monad.Reader
 import Control.Monad.Error.Class
-import Control.Monad.Except
 import Control.Monad.Trans.Control
+import Control.Monad.Trans.Either
```

Comment:
I'm fine with either way (no pun intended)... Initially I actually just changed all of `Utils` to use `ExceptT`, but then I realized that it would have resulted in a much larger changeset than necessary. But I'm happy to do that if you would prefer it.